lib/gracenote/search.rb

```diff
@@ -1,9 +1,15 @@
 module Gracenote
   module Search
     # @params mode [String] 'SINGLE_BEST' or 'SINGLE_BEST_COVER'
-    def search(artist: nil, album: nil, track: nil, mode: nil)
+    def search(artist: nil, album: nil, track: nil, mode: nil, range: nil)
       query("album_search", auth: auth, lang: lang) do |xml|
         xml.MODE { xml.text mode } if mode
+        if range
+          xml.RANGE {
```

Comment:
__[rubocop]__
* Avoid using `{...}` for multi-line blocks. [:ref](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)